packages/client/hmi-client/src/components/optimize/tera-intervention-policy-group-form.vue

```diff
@@ -0,0 +1,203 @@
+<template>
+	<div class="policy-group" :style="`border-left: 9px solid ${props.config.borderColour}`">
+		<div class="form-header">
+			<div class="header-items">
+				<InputText
+					v-if="isEditing"
+					v-model="policyName"
+					placeholder="Policy bounds"
+					@focusout="emit('update-self', { index, updatedConfig })"
+				/>
+				<h4 v-else>{{ props.config.name }}</h4>
+				<i
+					:class="{ 'pi pi-check i': isEditing, 'pi pi-pencil i': !isEditing }"
+					:style="'cursor: pointer'"
+					@click="onEdit"
+				/>
+			</div>
+			<div class="header-items">
+				<label for="active">Active</label>
+				<InputSwitch @change="emit('update-self', { index, updatedConfig })" v-model="active" />
+			</div>
+			<div>
+				<i
+					class="trash-button pi pi-trash"
+					:style="'cursor: pointer'"
+					@click="emit('delete-self', index)"
+				/>
+			</div>
+		</div>
+		<div class="input-row">
+			<div class="label-and-input">
```

Comment:
What about `<fieldset>`?
https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form#the_fieldset_and_legend_elements

Reply:
I tried this, but this added its own styling that messed up how the form looked :/